### PR TITLE
Doc: Fix inline comment invalid end char for a bunch of files

### PR DESCRIPTION
**Pull request by eri-trabiccolo**
## Description
covered:
```
root files
includes/abstracts
includes/achievements
includes/admin
```
I've also changed the phpcs.xml

Per #946

## How has this been tested?
n/a

## Screenshots 
n/a

## Types of changes
Documentation and coding standards, non-breaking

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

**Comment by eri-trabiccolo** [comment]
@thomasplevy re-opened.
Looks like I have a problem with `composer run-script check-cs`, it only parses files in the root directory, which is weird...
Is it something you can reproduce?

File: includes/admin/post-types/meta-boxes/class.llms.meta.box.voucher.export.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ public static function export() {
 
 			if ( 'vouchers' === $type || 'redeemed' === $type ) {
 
-				// export CSV
+				// Eexport CSV.
```
Comment (thomasplevy) [inline includes/admin/post-types/meta-boxes/class.llms.meta.box.voucher.export.php]: Spelling

```suggestion
				// Export CSV.
```

File: includes/admin/views/index.php [hunk 2]
```diff
@@ -1 +1 @@
-<?php // shhhh
+<?php // shhhh.
```
Comment (thomasplevy) [inline includes/admin/views/index.php]:
```suggestion
<?php // Shhhh.
```

File: phpcs.xml [hunk 3]
```diff
@@ -26,7 +26,8 @@
 		<exclude-pattern>assets/js/*.js</exclude-pattern>
 
 		<!-- @todo: Fix docs and comments to adhere to these rules -->
-		<exclude name="Squiz.Commenting.InlineComment.InvalidEndChar" />
+		<!-- exclude name="Squiz.Commenting.InlineComment.InvalidEndChar" /-->
```
Comment (thomasplevy) [inline phpcs.xml]:
```suggestion
```

**Review by thomasplevy — CHANGES_REQUESTED** [review]

**Pull request merged**